buildpack.toml

```diff
@@ -2,7 +2,8 @@ api = "0.5"
 
 [buildpack]
   id = "paketo-buildpacks/python"
-  name = "Paketo Python Buildpack"
+  name = "Paketo Python Buildpack (Plotly fork)"
+  version = "5.1.1b1"
```

Comment:
I think we should match the DE5 version here ?? It's not what we said?

Reply:
@x3rus Sorry I tagged the wrong PR. The correct one is: https://github.com/plotly/paketo-buildpacks_python/pull/12

I will close this one.